src/core/redux/ReduxReducer.js

```diff
@@ -2,27 +2,15 @@
  * @flow
  */
 
-import Immutable from 'immutable';
-
 import {
   combineReducers
 } from 'redux-immutable';
 
-const INITIAL_STATE :Map<*, *> = Immutable.fromJS({});
-
-// TODO - remove this from here once we start using reducers
-function appReducer(state :Map<*, *> = INITIAL_STATE, action :Object) :Map<*, *> {
-
-  switch (action.type) {
-
-    default:
-      return state;
-  }
-}
+import { reducer as catalogReducer } from '../../containers/catalog/CatalogReducer';
```

Comment:
if you change `CatalogReducer` to have `export default function reducer() {}`, then this can become

`import catalogReducer from '../../containers/catalog/CatalogReducer';`